docs/api-reference/core/linear-interpolator.md

```diff
@@ -10,12 +10,15 @@ See [View State Transitions](/docs/developer-guide/view-state-transitions.md) fo
 ```js
 import {LinearInterpolator} from '@deck.gl/core';
 
-new LinearInterpolator(['target', 'zoom']);
+new LinearInterpolator({transitionProps: ['target', 'zoom']});
 ```
 
 Parameters:
 
-* transitionProps (Array) - Array of prop names that should be linearly interpolated. Default `['longitude', 'latitude', 'zoom', 'bearing', 'pitch']`.
+- options (Object)
+  * `transitionProps` (Array, optional) - Array of prop names that should be linearly interpolated. Default `['longitude', 'latitude', 'zoom', 'bearing', 'pitch']`.
+  * `around` (Array, optional) - A point to zoom/rotate around, `[x, y]` in screen pixels. If provided, the location at this point will not move during the transition.
+  * `makeViewport` (Function, optional) - Called to construct a [viewport](/docs/api-reference/core/viewport.md), e.g. `props => new WebMercatorViewport(props)`. Must be provided if `around` is used.
```

Comment:
Requiring function props makes use of classes harder in declarative environments (JSON, pydeck etc). Nice if there was a default function, but maybe there is no natural default?

Reply:
There is no known use case where you want to specify a custom `transitionInterpolator` in declarative environments. It is used by:
- The default controllers in input handling and passed to `onViewStateChange`
- Response to other user input (e.g. "go to location on button click")